src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -960,6 +960,107 @@ default int getMaxEmotes()
+     * @return A {@link RestAction} containing a {@link GuildScheduledEvent} object, or {@code null} if an event with that ID does not exist in this guild,
+     *         or the currently logged in user account does not have access to it.
+     *
+     * @see #getScheduledEventById(long)
+     */
+    @Nullable
+    @CheckReturnValue
+    RestAction<GuildScheduledEvent> retrieveScheduledEventById(long id);
+
+    /**
+     * Returns a {@link GuildScheduledEvent} from cache by its ID
+     *
+     * @param id
+     *        The ID of the {@link GuildScheduledEvent}
+     *
+     * @return A {@link GuildScheduledEvent} object, or {@code null} if an event with that ID does not exist on this guild,
+     *        is not cached, or the currently logged in user account does not have access to it.
+     *
+     * @see #retrieveScheduledEventById(long)
+     */
+    @Nullable
+    GuildScheduledEvent getScheduledEventById(long id);
+
+    /**
+     * Returns a cached list of {@link GuildScheduledEvent GuildScheduledEvents} from this guild that is accessible to the
+     * currently logged in user.
+     *
+     * @return A list of {@link GuildScheduledEvent GuildScheduledEvent} objects
+     *
+     * @see #retrieveScheduledEvents()
+     */
+    @Nonnull
+    List<GuildScheduledEvent> getScheduledEvents();
+
+    /**
+     * Retrieves a list of {@link GuildScheduledEvent GuildScheduledEvents} from this guild that is accessible to the
+     * currently logged in user.
+     *
+     * @return A {@link RestAction} containing a list of {@link GuildScheduledEvent GuildScheduledEvent} objects
+     *
+     * @see #getScheduledEvents()
+     */
+    @Nonnull
+    @CheckReturnValue
+    RestAction<List<GuildScheduledEvent>> retrieveScheduledEvents();
+
+    /**
+     * Creates a new {@link GuildScheduledEvent}.
+     * <p><b>Requirements</b></p>
+     * <p>Events that are created are required to have a name, a location, and a start time. Depending on the
+     * type of location provided, an event will be of one of three different {@link GuildScheduledEvent.Type Types}:
+     * <ol>
+     *     <li>
+     *         <p><b>{@link GuildScheduledEvent.Type#STAGE_INSTANCE Type.STAGE_INSTANCE}</b></p>
+     *         <p>These events are set to take place inside of a {@link net.dv8tion.jda.api.entities.StageChannel}. The
+     *         following permissions are required in the specified stage channel in order to create an event there:
+     *          <ul>
+     *              <li>{@link net.dv8tion.jda.api.Permission#MANAGE_EVENTS Permission.MANAGE_EVENTS}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL Permission.MANAGE_CHANNEL}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#VOICE_MUTE_OTHERS Permission.VOICE_MUTE_OTHERS}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#VOICE_MOVE_OTHERS Permission.VOICE_MOVE_OTHERS}}</li>
+     *         </ul></p>
```

Comment:
You should not close `<p>` tags in javadocs.

```suggestion
     *         </ul>
```